src/styles/blocks/antisnake.scss

```diff
@@ -0,0 +1,23 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 1200px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  &__block {
+    display: inline-block;
+    color: #fff;
+    font-size: 100px;
+    line-height: 300px;
+    text-align: center;
+    height: 300px;
+
+    @each $number, $color in $colors {
+      &--#{$number} {
+        background-color: $color;
+      }
```

Comment:
Don't create any extra classes, try to implement this without creating any extra classes.

Hint: use nth-child pseudoclass.